<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Учет заявок на ремонт оргтехники</string>
  </property>
  <property name="windowIcon">
   <iconset theme="accessories-dictionary">
    <selectedon> ../media/logo.png</selectedon>
   </iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
background-color: rgb(4, 160, 255);
color: rgb(255, 255, 255);
border: 0px;
}
QWidget
{font: &quot;Comic Sans MS&quot;;}
</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(4, 160, 255, 255), stop:1 rgba(231, 250, 191, 255));}

</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="labelAlignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Учет заявок на ремонт оргтехники</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>media/logo.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Avtorisation">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>200</number>
       </property>
       <property name="topMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>200</number>
       </property>
       <property name="bottomMargin">
        <number>50</number>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>24</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>АВТОРИЗАЦИЯ</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Логин</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LoginField"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Пароль</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PasswordField"/>
         </item>
         <item>
          <widget class="QPushButton" name="SignInButton">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Войти</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ErrorField">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Zakazchik">
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>190</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Заказчик</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>120</y>
         <width>103</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton {background-color: rgb(255,125,150);
border: 5x;}</string>
       </property>
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Manager">
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>320</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>менеджер</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>250</y>
         <width>103</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton {background-color: rgb(255,125,150);
border: 5x;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Operator">
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>250</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>оператор</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>180</y>
         <width>103</width>
         <height>36</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pushButton {background-color: rgb(255,125,150);
border: 5x;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Master">
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>240</y>
         <width>66</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Master</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="SignOutButton">
     <property name="text">
      <string>выход</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
